arch/sim/Kconfig

```diff
@@ -134,10 +134,7 @@ endif
 config SIM_NETDEV
 	bool "Simulated Network Device"
 	default y
-	depends on NET_ETHERNET
-	select ARCH_HAVE_NETDEV_STATISTICS
-	select SCHED_LPWORK
-	select SIM_WALLTIME
+	depends on (NET_ETHERNET && SIM_WALLTIME && ARCH_HAVE_NETDEV_STATISTICS && SCHED_LPWORK)
```

Comment:
ARCH_HAVE_NETDEV_STATISTICS is designed to be selected, and the most place select SCHED_LPWORK, not depend on SCHED_LPWORK. so it's better to change:
	depends on NET_ETHERNET && SIM_WALLTIME
	select ARCH_HAVE_NETDEV_STATISTICS
	select SCHED_LPWORK

Reply:
Ok I updated to this